#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_config.dpatch by Denis V. Sirotkin <fechiny@gmail.com>
## Original changes was done by Tom Huckstep <tom-debian-polipo@jaguarpaw.co.uk>
##
## DP: Change description

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
	echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
	exit 1;;
esac

exit 0

@DPATCH@

--- polipo-1.0.1.orig/config.sample
+++ polipo-1.0.1/config.sample
@@ -1,8 +1,7 @@
 # Sample configuration file for Polipo. -*-sh-*-
 
-# You should not need to use a configuration file; all configuration
-# variables have reasonable defaults.  If you want to use one, you
-# can copy this to /etc/polipo/config or to ~/.polipo and modify.
+# You should not need to edit this configuration file; all configuration
+# variables have reasonable defaults.
 
 # This file only contains some of the configuration variables; see the
 # list given by ``polipo -v'' and the manual for more.
@@ -17,11 +16,11 @@
 # proxyAddress = "::0"        # both IPv4 and IPv6
 # proxyAddress = "0.0.0.0"    # IPv4 only
 
-# If you do that, you'll want to restrict the set of hosts allowed to
-# connect:
-
-# allowedClients = "127.0.0.1, 134.157.168.57"
-# allowedClients = "127.0.0.1, 134.157.168.0/24"
+# If you are enabling 'proxyAddress' above, then you want to enable the
+# 'allowedClients' variable to the address of your network, e.g.
+# allowedClients = 127.0.0.1, 192.168.42.0/24
+                                            
+# allowedClients = 127.0.0.1              
 
 # Uncomment this if you want your Polipo to identify itself by
 # something else than the host name:
